SwiftStyleKitSample/StyleKitSample/Style.swift

```diff
@@ -0,0 +1,348 @@
+        didSet {
+            var info = [String: UIButton]()
+            for button in StyleButton {
+                if let styleTag = button.styleTag {
+                    info[styleTag] = button
+                }
+            }
+            self.style(withButtonsAndStyles: info)
+        }
+    }
+    
+    @IBOutlet var StyleTextField: [UITextField]! {
+        didSet {
+            var info = [String: UITextField]()
+            for textField in StyleTextField {
+                if let styleTag = textField.styleTag {
+                    info[styleTag] = textField
+                }
+            }
+            self.style(withTextFieldsAndStyles: info)
+        }
+    }
+    
+    //Mark: Serialize
+    
+    func configurationStyleURL() -> NSURL? {
+        let documentsDirPath = urlForFileInDocumentsDirectory("Style.json")
+        if NSFileManager.defaultManager().fileExistsAtPath(documentsDirPath.path!)   {
+            return documentsDirPath
+        }
+
+        let bundlePath = NSBundle.mainBundle().pathForResource("Style", ofType: "json")
+        return NSURL(fileURLWithPath: bundlePath!)
+    }
+    
+    func urlForFileInDocumentsDirectory(fileName: String) -> NSURL {
+        let docsDir = NSFileManager.defaultManager().URLsForDirectory(.DocumentDirectory, inDomains: .UserDomainMask)[0]
+        return docsDir.URLByAppendingPathComponent(fileName)
+    }
+    
+    func serialize() {
+        
+        let stylePath = configurationStyleURL()!
+        
+        do {
+            
+            let data = try NSData(contentsOfURL: stylePath, options: NSDataReadingOptions.DataReadingMappedIfSafe)
+            let json = try NSJSONSerialization.JSONObjectWithData(data, options: .AllowFragments)
+            
+            if let items = json["Fonts"] as? [String: String] {
+                fonts = items
+            }
+            
+            if let colorDict = json["Colors"] as? [String: [String: Int]] {
+                for (colorKey, components) in colorDict {
+                    if let red = components["red"],
+                        let green = components["green"],
+                        let blue = components["blue"],
+                        let alpha = components["alpha"] {
+                            colors[colorKey] = UIColor(red: CGFloat(red)/255, green: CGFloat(green)/255, blue: CGFloat(blue)/255, alpha: CGFloat(alpha))
+                    }
+                }
+            }
+            
+            imageNames = json["Images"] as? [String: String]
+            
+            if let labelDict = json["Labels"] as? [String: [String:AnyObject]] {
+                for (labelKey, specification) in labelDict {
+                    labelStyles[labelKey] = serializeLabelSpec(specification)
+                }
+            }
+            
+            if let buttonDict = json["Buttons"] as? [String: [String:AnyObject]] {
+                for (labelKey, specification) in buttonDict {
```

Comment:
You're right. I was copying from the previous block so I was not aware of the naming. I'll fix it.